spec/author_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../classes/item/item'
+require_relative '../classes/author/author'
+
```

Comment:
Great work on all the specs.  Unit test are descriptive and test all public methods 👏🏾 

![image](https://github.com/ahmidou09/Catalog-of-my-things/assets/38088818/a123d7ae-ed2e-418d-8463-6ed5438eb347)